R/execute_pipeline.R

```diff
@@ -303,28 +313,28 @@ run_targets_pipeline <- function(
                                cell_cycle_score_tbl,
                                annotation_label_transfer_tbl,
                                doublet_identification_tbl),
-                 iteration = "list"),
+                 iteration = "list")
       
       # pseudobulk preprocessing for each sample 
-      tar_target(create_pseudobulk_sample, create_pseudobulk(preprocessing_output_S, 
-                                                             assays = "SCT", 
-                                                             cell_type_annotation_column, 
-                                                             x = c(sampleName, cellAnno)), 
-                 pattern = map(preprocessing_output_S), 
-                 iteration = "list"),
-      
-      tar_target(pseudobulk_merge_all_samples, pseudobulk_merge(create_pseudobulk_sample, 
-                                                                assays = "RNA", 
-                                                                x = c(sampleName)), 
-                 iteration = "list"),
-      
-      tar_target(calc_UMAP_dbl_report, calc_UMAP(input_read), 
-                 pattern = map(input_read), 
-                 iteration = "list"), 
-      tar_target(variable_gene_list, find_variable_genes(input_read, 
-                                                         empty_droplets_tbl), 
-                 pattern = map(input_read, empty_droplets_tbl), 
-                 iteration = "list")
+      # tar_target(create_pseudobulk_sample, create_pseudobulk(preprocessing_output_S,
+      #                                                        assays = "SCT",
+      #                                                        cell_type_annotation_column,
+      #                                                        x = c(sampleName, cellAnno)),
+      #            pattern = map(preprocessing_output_S),
+      #            iteration = "list"),
+      # 
+      # tar_target(pseudobulk_merge_all_samples, pseudobulk_merge(create_pseudobulk_sample,
+      #                                                           assays = "RNA",
+      #                                                           x = c(sampleName)),
+      #            iteration = "list"),
+      # 
+      # tar_target(calc_UMAP_dbl_report, calc_UMAP(input_read),
+      #            pattern = map(input_read),
+      #            iteration = "list"),
+      # tar_target(variable_gene_list, find_variable_genes(input_read,
+      #                                                    empty_droplets_tbl),
+      #            pattern = map(input_read, empty_droplets_tbl),
```

Comment:
Are we sure @susansjy22 does not need this? Maybe she commented temporarily. @susansjy22 can you please comment on this?

Reply:
These functions are for the creation of reports. However, they need to be refined to be reproducible for different input data. I temporarily commented them out for testing purposes.

Reply:
@myushen then please leave them commented @susansjy22 will fix this soon.

Reply:
I commented `pseudobulk` targets for now since census data only needs preprocessing output. There might be a solution to skip running certain targets instead of commenting?